.gitignore

```diff
@@ -6,6 +6,8 @@ node_modules/
 
 # Compiled files
 support/output-targets/custom-elements/bundles/
+support/output-targets/calcite-components-react/src/components.ts
+support/output-targets/calcite-components-react/src/react-component-lib/
```

Comment:
I just pulled the branch to try out `npm link`ing this into a project and testing things out, but I wasn't able to execute `npm run build` in `support/output-targets/calcite-components-react` since the `src/components.ts` file is missing, am I missing a step?

Reply:
yes, you need to generate that file by running a build in the main `calcite-components` repo

Reply:
@jgibson02 any luck?

Reply:
There seemed to be some npm weirdness on my end so I had to reinstall Node, but I've generally got it working in Velocity, noticing some oddities with the attributes on the rendered DOM elements and handling of `={true}` / `={false}`. Some components seem to handle the props while others don't, but I'm not sure.

e.g. `disableSpacing` => `disablespacing="true"` doesn't seem to work, but `labelTicks` has the same behavior and works
![image](https://user-images.githubusercontent.com/5069711/112536385-a4734680-8d6a-11eb-998d-8c60e2b28c57.png)

Reply:
Interesting, I can take a look at this later on after I finish up some 9.1 stuff. Thanks for checking this out 👍🏻